src/main/java/seedu/souschef/model/favourite/Favourites.java

```diff
@@ -39,11 +44,9 @@ public Favourites(Name name, Difficulty difficulty, CookTime cooktime, Set<Tag>
         this.name = name;
         this.cookTime = cooktime;
         this.difficulty = difficulty;
+        this.instructions.addAll(instructions);
```

Comment:
Class is redundant, just a copy of the recipe class.
Try to understand what you need to implement and add in the change.